src/gitgraph.js

```diff
@@ -317,22 +359,18 @@
     var self = this.gitgraph;
     var isOut = true;
 
-    // Fix firefox MouseEvent
-    if ( typeof InstallTrigger !== "undefined" )/* == (is Firefox) */ { 
-      event.offsetX = event.offsetX ? event.offsetX : event.layerX;
-      event.offsetY = event.offsetY ? event.offsetY : event.layerY;
-      event.x = event.x ? event.x : event.clientX;
-      event.y = event.y ? event.y : event.clientY;
-    }
-
-    function showCommitTooltip () {
+    function showCommitTooltip (commit) {
       self.tooltip.style.left = event.x + "px"; // TODO Scroll bug
       self.tooltip.style.top = event.y + "px";  // TODO Scroll bug
-      self.tooltip.textContent = commit.sha1 + " - " + commit.message;
+      if (!(self.template.commit.tooltipHTMLFormatter === null)) {
+        self.tooltip.innerHTML = self.template.commit.tooltipHTMLFormatter(commit);
```

Comment:
@terhechte > I see you also added the ability to format tooltips. That's nice, I'll check and document that too :+1: